R/rfh_load_scripts.R

```diff
@@ -1,92 +0,0 @@
-#' Load all R scripts in the factory
-#'
-#' This helper function will load all `.R` scripts within folders `scripts/` and
```

Comment:
Can we re-add this file with its test without damaging anything?

Reply:
closed in https://github.com/reconhub/reportfactory/pull/68/commits/aebd94971e06de3738dc2bd11a316cf142aebca9